src/main/java/com/google/sps/managers/IDataManager.java

```diff
@@ -20,68 +20,68 @@
 
 public interface IDataManager {
   /**
-    * Creates or Replaces an entity of a single User data in datastore
-    * @param user a User variable representing a single instance of a user.
+  * Creates or Replaces an entity of a single User data in datastore
+  * @param user a User variable representing a single instance of a user.
   */
   public void createOrReplaceUser(User user);
 
   /**
-    * Retrieves a single user entity from the datastore.
-    * @param userID the unique code used to identify this specific user.
-    * @return a user object with the properties specified in the Builder.
+  * Retrieves a single user entity from the datastore.
+  * @param userID the unique code used to identify this specific user.
+  * @return a user object with the properties specified in the Builder.
   */
   public User retrieveUser(String userID);
 
   /**
-    * Creates or Replaces the static data of a single game in datastore as an entity
-    * @param game an Game variable representing a single instance of a game
+  * Creates or Replaces the static data of a single game in datastore as an entity
+  * @param game an Game variable representing a single instance of a game
   */
   public void createOrReplaceGame(Game game);
 
   /**
-    * Retrieves a single game entity from the datastore.
-    * @param gameID the unique code used to identify this specific game.
-    * @return a Game object with the properties specified in the Builder.
+  * Retrieves a single game entity from the datastore.
+  * @param gameID the unique code used to identify this specific game.
+  * @return a Game object with the properties specified in the Builder.
   */
   public Game retrieveGame(String gameID);
   
   /**
-    * Creates the static data of a single stage in datastore as an entity
-    * @param game an Stage variable representing a single instance of a stage.
+  * Creates the static data of a single stage in datastore as an entity
+  * @param game an Stage variable representing a single instance of a stage.
```

Comment:
* @param **stage** an Stage variable representing a single instance of a **S**tage.*